modal-js/src/volume.ts

```diff
@@ -0,0 +1,43 @@
+import { VolumeFsVersion, ObjectCreationType } from "../proto/modal_proto/api";
+import { client } from "./client";
+import { environmentName as configEnvironmentName } from "./config";
+import { ClientError, Status } from "nice-grpc";
+import { NotFoundError } from "./errors";
+
+/** Options for `Volume.fromName()`. */
+export type VolumeFromNameOptions = {
+  environment?: string;
+  createIfMissing?: boolean;
+  version?: 1 | 2;
```

Comment:
Is the `version` supposed to be on the Options when looking up the volume, or is it part of the metadata of the volume that you get after looking it up?

Reply:
Yeah good point, discussed with @freider now and I think we'll just remove it from the API here! wdyt?